mobile.html

```diff
@@ -273,7 +305,50 @@ <h2 class="project-title3">Multi-Post Stories</h2>
         <img src="icons/back2.png" alt="" class="back2">
         <img src="icons/back3.png" alt="" class="back3">
         <img src="icons/back4.png" alt="" class="back4">
+        <img src="icons/desktop1.png" alt="" class="desk1">
+        <img src="icons/desktop2.png" alt="" class="desk2">
+        <img src="icons/desktop3.png" alt="" class="desk3">
+        <img src="icons/desktop4.png" alt="" class="desk4">
     </div>
+    <section class="contact-form">
+        <div class="contact-me">
+            <h2 class="contact-us">contact us</h2>
+            <h2 class="chat">let's chat</h2>
+            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        </div>
+        <form action="https://formspree.io/f/xjvjloyy" method="POST" class="form">
+          <input id="input" type="text" required placeholder="text input" maxlength="30" name="name">
+          <input type="email" required placeholder="myemail@gmail.com" name="email" id="email">
+          <textarea name="message" id="message" cols="30" rows="10" placeholder="Enter text here" required></textarea>
```

Comment:
- [ ] **REQUIRED:** Your work is admirable, but there are some missing requirements here. Please add the maxlength property for the message and set it to 500